src/picturepark-sdk-v2-angular/projects/picturepark-sdk-v2-angular-ui/src/lib/features-module/content-download-dialog/content-download-dialog.component.ts

```diff
@@ -99,6 +99,7 @@ export class ContentDownloadDialogComponent extends DialogBaseComponent implemen
         .subscribe(output => this.setSelection(output));
     } else {
       if (this.data.contents.every(content => content.outputs)) {
+        // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
         const outputs = flatMap(this.data.contents, content => content.outputs!);
```

Comment:
try to pass an empty array in case the outputs were undefined and delete the disable